### PR TITLE
Re-add check for valid Julia path

### PR DESCRIPTION
**Pull request by pfitzseb**
We lost that check in one of the path finding rewrites.

image

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -20,6 +20,8 @@ const config = {
     externals: {
         vscode: 'commonjs vscode', // the vscode-module is created on-the-fly and must be excluded. Add other modules that cannot be webpack'ed, 📖 -> https://webpack.js.org/configuration/externals/
         'applicationinsights-native-metrics': 'applicationinsights-native-metrics',
+        '@opentelemetry/instrumentation': '@opentelemetry/instrumentation',
+        '@azure/opentelemetry-instrumentation-azure-sdk': '@azure/opentelemetry-instrumentation-azure-sdk'
```
Comment (pfitzseb) [inline webpack.config.js:24]: This is unrelated and just gets rid of two errors/warnings when building.

**Review by davidanthoff — APPROVED** [review]
